src/multiplayer.h

```diff
@@ -137,7 +145,7 @@ class MultiplayerObject : public virtual PObject
         const char* name;
 #endif
         void* ptr;
-        int64_t prev_data;
+        void* prev_data;
```

Comment:
This most likely will break things in a 32bit build. As this field is used in different ways (a union of a double, uint64 and pointer is more correct)